lib/mutant/repository.rb

```diff
@@ -20,17 +20,9 @@ def call(subject)
 
     # Diff between two objects in repository
     class Diff
-      include Adamantium, Concord.new(:from, :to)
+      include Adamantium, Anima.new(:config, :from, :to)
```

Comment:
Do you think the threshold for switching from Concord to Anima is 3 parameters?

I normally wouldn't have a problem with 3 params in a constructor, 4 and I would probably use Anima though (or rethink my object because 4 is probably pushing it).